hw/top_earlgrey/chip_earlgrey_asic.core

```diff
@@ -12,6 +12,8 @@ filesets:
       - lowrisc:systems:padring
       - "!fileset_partner ? (lowrisc:systems:ast)"
       - "fileset_partner ? (partner:systems:ast)"
+      - "!fileset_partner ? (lowrisc:systems:scan_role_pkg)"
```

Comment:
minor suggestion, could we group the dependencies such that..

```
"fileset_partner ? ..."
"fileset_partner ? ..."
"!fileset_partner ? ..."
"!fileset_partner ? ..."

Reply:
ah yeah, let's do that.